lvk/LVK.h

```diff
@@ -471,6 +471,7 @@ enum Format : uint8_t {
   Format_Z_UN24,
   Format_Z_F32,
   Format_Z_UN24_S_UI8,
+  Format_Z_F32_S_UI8,
```

Comment:
Let's mark this as Mac-specific.

Reply:
I suppose any platform can create it if it's supported by GPU. VK_FORMAT_D32_SFLOAT_S8_UINT exists in the spec.